### PR TITLE
fix(calculator): make more responsive

### PR DESCRIPTION
**Pull request by wintersummermint**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?
Old -> Proposed 
LfhmNrkGuC

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

(insert description here)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ report*.json
 package-lock.json
 frontend/package-lock.json
 .vs
+frontend/package-lock.json
```
Comment (seiyria) [inline .gitignore:14]: do not just add things to gitignore.

**Review by seiyria — APPROVED** [review]

**Review by seiyria — CHANGES_REQUESTED** [review]
lint needs to be fixed

**Comment by wintersummermint** [comment]
Didn't detect diff on removing trailing space, so deleted newline and trail so it can be recognized

**Comment by wintersummermint** [comment]
all good now i suppose

**Review by seiyria — APPROVED** [review]

**Pull request merged**